.jenkins/pytorch/win-test-helpers/build_pytorch.bat

```diff
@@ -45,7 +45,7 @@ set CUDA_PATH_V9_2=%CUDA_PATH%
 goto cuda_build_common
 
 :cuda_build_10
-call "C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Auxiliary\Build\vcvarsall.bat" x64 -vcvars_ver=14.11
+call "C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Auxiliary\Build\vcvarsall.bat" x64 -vcvars_ver=%VC_VERSION%
```

Comment:
Actually, we can make 2017 and BuildTools as parameters too.
`VS_VERSION`: 2017
`VS_PRODUCT`: BuildTools